o/opentracing/redhat_ubi8/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM registry.access.redhat.com/ubi8/ubi:latest
```

Comment:
Please replace with
FROM adoptopenjdk/openjdk8-openj9:ubi
it is preconfigured to use openj9 JVM with openjdk1.8
also, change 
RUN dnf install -y gcc gcc-c++ cmake make git curl \java-1.8.0-openjdk.ppc64le java-1.8.0-openjdk-devel.ppc64le 
to
RUN dnf install -y gcc gcc-c++ cmake make git curl

Reply:
I have updated the Dockerfile as required.